lib/puppet/provider/package/pear.rb

```diff
@@ -36,7 +36,7 @@ def self.pearlist(hash)
           pearhash[:provider] = :pear
           pearhash
         end
-      end.reject { |p| p.nil? }
+      end.reject(&:nil?)
```

Comment:
I was wondering if this could actually be replaced with `end.compact`, but was too scared!